python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -4,40 +4,41 @@
 from pathlib import Path
 from typing import List, Optional, Union
 
-from dagster._annotations import experimental, public
+from dagster._annotations import deprecated, experimental, public
 
 from .core.resources_v2 import DbtCliResource
 from .errors import DagsterDbtManifestNotPreparedError, DagsterDbtProjectNotFoundError
 
-logger = logging.getLogger("dagster-dbt.artifacts")
+logger = logging.getLogger("dagster-dbt.project")
 
 
 @experimental
-class DbtArtifacts:
+class DbtProject:
```

Comment:
I had this as `DagsterDbtProject` before and have been going back and forth

Reply:
I guess they have just `Project`: https://github.com/dbt-labs/dbt-core/blob/c6c0c79216a3c8032d5ea5ed608ca2ef7bdacc9a/core/dbt/contracts/project.py#L188